src/app/newcomers/newcomers.component.html

```diff
@@ -0,0 +1,58 @@
+<div class="container-fluid mt-5">
+  <div class=" container box logos mx-auto d-block">
+    <div class="row text-center">
+      <h1 class="mx-auto mb-5"> Welcome to Systers Open Source!</h1>
+      <p>
+        Systers is an international Anita B.org community encouraging the participation of all women involved in technology.
+      </p>
+      <p>
+        We are committed to providing a friendly, safe, and welcoming environment for all, regardless of gender, sexual orientation,
+        ability, ethnicity, socioeconomic status, work experience, or religion (or lack thereof).
+      </p>
+      <p class="mb-5">
+        We believe in inclusivity, diversity, and transparency. We value open source citizenship and collaborative involvement.
+      </p>
+    </div>
+    <div class="row text-center">
+      <img class="container-fluid mb-5" src="/assets/img/full_map.png">
+    </div>
+    <div>
+      <p class="text-center">
+        At Systers, we welcome contributions of all forms. Non-coders are welcomed to join the community!
+        <br> Apart from code you can contribute in any of the following ways:
+      </p>
+    </div>
+    <div class="container box logos mx-auto d-block">
+      <div class="row text-center justify-content-center">
+        <div class="card col-sm-3 mt-3 mb-3 ml-3 mr-3 pt-2" *ngFor="let det of method_details">
+          <img class="card-img-top" src="{{det.image}}" alt="{{det.methodName}}">
+          <div class="card-body">
+            <h5 class="card-title">{{det.methodName}}</h5>
+            <button type="button" class="btn btn-primary waves-light mx-auto" (click)="style.show()" mdbWavesEffect>View Details</button>
+            <!-- Modal -->
+									<div mdbModal #style="mdb-modal" class="modal fade" id="centralModalSuccess" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"
+                  aria-hidden="true">
+                <div class="modal-dialog modal-notify modal-success" role="document">
+                  <div class="modal-content">
+                    <div class="modal-header" style="background-color: #8d2165;">
```

Comment:
bg color move to scss

Reply:
@ancyp I have shifted the style attribute for the <p> to the scss file, but if I put the one for the modal-header in the scss file, the colour of the modal changes to green, which is the default colour for the modals headers in mdBootstrap. And since the project.component.scss file is imported before the mdBootstrap default stylesheet, the colour which shows up eventually is green. However in the case, I put it up as the style attribute, it overrides all the existing properties and thus the colour of the modal is maroon. Therefore, in the case this has to be shifted to the scss file, I will have to study in-depth about what is the order in which Angular adds the scss files. Let me know, if I can make this into a separate issue and we merge the PR?

Reply:
@ancyp I have shifted the style attribute form the <p> tag to the scss file, but for the model-header class, the style property for background colour when put in the scss file, is overridden by the background-color property (green) of model-header class in the mdBootstrap stylesheet file, since perhaps the newcomers.component.scss file is accessed before the the mdBootstrap default css file and thus the green colour replaces the maroon one mentioned in the css file. 
In the case that I add the same statement in the style attribute, it overrides all the existing scss imported attributes and thus the modal-header appears maroon. So in the case we want to remove the style attribute of the modal-header, one would need to go into the depth and analyse how Angular is loading its files. So can we do with the current commit or can we create a separate issue to solve this one, later?